docs/v3.x/content-api/integrations.md

```diff
@@ -0,0 +1,22 @@
+# Integrations
+
+Strapi generates an API for you to access your content. But how can you connect a React, Ruby, Gatsby application to it?
+It is important to know what is an API.
+
+## What is an API?
+
+API is the acronym for Application Programming Interface, which is a software intermediary that allows two applications to talk to each other.
+In case you want to connect a React application with Strapi, we say that React is the client and Strapi the system. Indeed, React will communicate to Strapi, by making HTTP requests. Strapi will then give a response back to your client.
+
+If your Strapi application contains restaurants and you want to list them in your React application, all you need to do is to make an HTTP request to Strapi which will take care to give you a response containing your restaurants.
+
+The [API Endpoints](../content-api/api-endpoints.html#api-endpoints) documentation will give you all the keys in hand to interact with your Strapi API.
+
+## Get started
+
+Today, any programming language has an HTTP client allowing you to execute requests to an API and therefore interact with it. Javascript has [Axios](https://github.com/axios/axios), [Fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API), Ruby has [Faraday](https://github.com/lostisland/faraday), [HTTParty](https://github.com/jnunemaker/httparty), Python has [Requests](https://requests.readthedocs.io/en/master/) etc...
+
+Integrate Strapi with a multitude of frameworks, frontend or backend programming languages just below.
+
+<IntegrationLinks>
```

Comment:
Just looking at the layout of this page, I'm wondering if we should move the buttons to the top. What do you think @lauriejim (included screenshot below)

Integrations: 
![image](https://user-images.githubusercontent.com/8593673/89308347-00942980-d627-11ea-8768-b378c7a58012.png)

Deployment:
![image](https://user-images.githubusercontent.com/8593673/89308402-11dd3600-d627-11ea-9aab-39bf86076ed9.png)

Reply:
I placed the buttons at the bottom because I want to be sure that they understand what is an API first before jumping in

Reply:
Also, I'm planning to just make javascript integrations for the moment but after that, a lot more buttons will come and they will never see the API definition

Reply:
Ah good point, the stuff on the deployment page is more like steps but yeah I agree having the definition first certainly does make sense. It's a shame that vuepress doesn't allow us to change the width of the content so we can cut out some of the horizontal scrolling in the code blocks :/

Reply:
So I finished the React, Vue.js, Next.js, Nuxt.js and Gatsby page. Should I add everything in this PR for you to review?
@derrickmehaffy @lauriejim @alexandrebodin @soupette?

Reply:
@Mcastres yeah probably easier here to do one single PR